app/presenters/users/approvals_presenter.rb

```diff
@@ -26,6 +26,33 @@ def gon
       gon
     end
 
+    def input_options
+      if apps_page?
+        { placeholder: "App name", class: "validate devs_typeahead", required: true }
+      else
+        { placeholder: "email@email.com", class: "validate", required: true }
+      end
+    end
+
+    def pending_friends
```

Comment:
Does this method do something similar to [to_sentence](https://apidock.com/rails/Array/to_sentence) ?